klog.go

```diff
@@ -898,6 +898,18 @@ func LogToStderr(stderr bool) {
 	logging.toStderr = stderr
 }
 
+// Determine whether the given path file/folder exists
+func exists(path string) bool {
+	_, err := os.Stat(path)
+	if err != nil {
+		if os.IsExist(err) {
+			return true
+		}
+		return false
+	}
+	return true
+}
+
```

Comment:
Checking to see if the file exists each time can be costly in terms of I/O consumption.

Using file watch is a better choice.